src/commands/integrations/init/yaml.ts

```diff
@@ -0,0 +1,410 @@
+        eslintConfig: {
+          root: true,
+          extends: ["@prismatic-io/eslint-config-spectral"],
+        },
+        dependencies: {
+          "@prismatic-io/spectral": "*",
+        },
+        devDependencies: {
+          "@prismatic-io/eslint-config-spectral": "2.0.1",
+          "@types/jest": "29.5.12",
+          "copy-webpack-plugin": "12.0.2",
+          jest: "29.7.0",
+          prettier: "3.4.2",
+          "ts-jest": "29.1.2",
+          "ts-loader": "9.5.1",
+          typescript: "5.5.3",
+          webpack: "5.91.0",
+          "webpack-cli": "5.1.4",
+        },
+      });
+    } finally {
+      process.chdir(cwd);
+    }
+  }
+
+  static async invoke(args: { [K in keyof typeof this.flags]+?: unknown }, config: Config) {
+    await GenerateIntegrationFromYAMLCommand.run(toArgv(args), config);
+  }
+}
+
+export function createFlowInputsString(action: ActionObjectFromYAML): string {
+  const inputs = action.inputs ?? {};
+  let resultString = "";
+
+  Object.entries(inputs).forEach(([key, input]) => {
+    let currentInputString = "";
+
+    if (input.type === "complex") {
+      let shouldWrap = true;
+      let isObject = false;
+
+      const castInputs = input.value as ValidComplexYAMLValue;
+      if (!castInputs || castInputs.length === 0) {
+        shouldWrap = false;
+        currentInputString = `${JSON.stringify(input.value)},`;
+      } else {
+        castInputs.forEach((element) => {
+          if (element.name) {
+            isObject = true;
+            currentInputString += `${element.name.value}: ${wrapValue(element.value)},`;
+          } else {
+            currentInputString += `${wrapValue(element.value)},`;
+          }
+        });
+      }
+
+      if (shouldWrap) {
+        currentInputString = isObject ? `{${currentInputString}},` : `[${currentInputString}],`;
+      }
+    } else {
+      currentInputString += `${wrapValue(input.value as ValidYAMLValue)},`;
+    }
+
+    if (action.isTrigger) {
+      resultString += `\n${key}: { value: ${currentInputString} },`;
+    } else {
+      resultString += `\n${key}: ${currentInputString}`;
+    }
+  });
+
+  return resultString;
+}
+
+function formatFlowForEJS(flow: FlowObjectFromYAML) {
+  const { name, description, isSynchronous, endpointSecurityType } = flow;
+  const steps: Array<ActionObjectFromYAML> = [];
+  let trigger: ActionObjectFromYAML | undefined;
+
+  flow.steps.forEach((step) => {
+    step.formattedInputs = createFlowInputsString(step);
+
+    if (step.isTrigger) {
+      trigger = step;
+    } else {
+      steps.push(step);
+    }
+  });
+
+  if (!trigger) {
+    // @TODO - we can maybe make this more forgiving
+    throw "No trigger found on flow.";
+  }
+
+  return {
+    key: camelCase(name),
+    flow: {
+      name,
+      stableKey: kebabCase(name),
+      description,
+      isSynchronous,
+      endpointSecurityType,
+    },
+    trigger,
+    steps,
+  };
+}
+
+function determinePermissionAndVisibilityType(
```

Comment:
Note - This definitely needs to be double-checked for correctness. Not exactly sure how to boil down the combination of meta & orgOnly into a permission & visibility string.